sqlglot/dialects/oracle.py

```diff
@@ -69,6 +69,10 @@ class Parser(parser.Parser):
 
         FUNCTION_PARSERS: t.Dict[str, t.Callable] = {
             **parser.Parser.FUNCTION_PARSERS,
+            "JSON_ARRAY": lambda self: self._parse_json_array(
+                exp.JSONArray,
+                expressions=self._parse_csv(lambda: self._parse_format_json(self._parse_bitwise())),
+            ),
```

Comment:
FYI moved this in Oracle because MySQL also has a `JSON_ARRAY` function with different syntax, and so it would be parsed incorrectly if this was in base. Perhaps we can pop the entry off of MySQL's function parsers instead?